x-pack/test/security_solution_cypress/cypress/tasks/api_calls/kibana_advanced_settings.ts

```diff
@@ -9,6 +9,10 @@ import { SECURITY_SOLUTION_SHOW_RELATED_INTEGRATIONS_ID } from '@kbn/management-
 import { ENABLE_EXPANDABLE_FLYOUT_SETTING } from '@kbn/security-solution-plugin/common/constants';
 import { rootRequest } from '../common';
 
+const ADVANCED_SETTINGS = {
```

Comment:
I thought I removed this, but @davismcphee we should be able to remove this correct?

Reply:
I have removed it.